bin/jenkinsGet.sh

```diff
@@ -31,9 +34,9 @@ function jenkinsGet (){
         PRODUCT_DESCRIPTOR=$(basename "$BRANCH_DECODED")
     fi
 
-    UNPACKED="/mnt/data/axonIvyProducts/${ARTIFACT}_$REVISION-$PRODUCT_DESCRIPTOR"
+    UNPACKED="${DATA_DIRECTORY}/${ARTIFACT}_$REVISION-$PRODUCT_DESCRIPTOR"
     echo "Extracting to $UNPACKED"
     unzip -q "/tmp/$ZIP" -d "$UNPACKED"
-    $(nemo "$UNPACKED" & )
+    gtk-launch "$(xdg-mime query default inode/directory)" "$UNPACKED"
```

Comment:
I already spent quite a while to find out a generic command without success, thanks a lot for sharing